src/systemcmds/tests/test_crypto.cpp

```diff
@@ -71,6 +71,67 @@ static uint32_t test_nonce[64 / 4] = {
 	0x9fb7a36e, 0xdafd7f9d, 0x1848d166, 0x398b3c9d, 0xcae67c98, 0x9b1d77e3, 0xbffed534, 0x02dffc3a,
 };
 
+
+const uint8_t test_rsa_pubkey_der[294] = {
+	0x30, 0x82, 0x01, 0x22, 0x30, 0x0d, 0x06, 0x09, 0x2a, 0x86, 0x48, 0x86, 0xf7, 0x0d, 0x01, 0x01,
+	0x01, 0x05, 0x00, 0x03, 0x82, 0x01, 0x0f, 0x00, 0x30, 0x82, 0x01, 0x0a, 0x02, 0x82, 0x01, 0x01,
+	0x00, 0xaa, 0x43, 0x61, 0x20, 0xc7, 0x04, 0x42, 0x1b, 0x55, 0xaa, 0x5f, 0x59, 0x70, 0xf0, 0xe8,
+	0x47, 0x9b, 0x95, 0xd6, 0x5a, 0xbf, 0x5f, 0x53, 0xd3, 0x97, 0xb8, 0x5e, 0xf0, 0x36, 0x0f, 0x41,
+	0x21, 0x9f, 0x62, 0x14, 0xe4, 0x56, 0xa2, 0x9f, 0xe3, 0x19, 0xbe, 0x27, 0xb2, 0x60, 0x18, 0x4f,
+	0xed, 0xb5, 0x50, 0x62, 0xb5, 0xc6, 0x61, 0xf7, 0xf1, 0x55, 0x4d, 0xd1, 0xf7, 0x46, 0x57, 0xf6,
+	0x81, 0x60, 0x3a, 0xe7, 0x93, 0x41, 0x04, 0xff, 0xe7, 0xc2, 0xd9, 0x59, 0x0d, 0xe4, 0x88, 0x8c,
+	0xf9, 0xf7, 0x46, 0xb4, 0xcc, 0xe2, 0x78, 0xc2, 0x13, 0x4b, 0xb4, 0xd1, 0x0a, 0x14, 0xca, 0x35,
+	0x25, 0x3f, 0x3c, 0x4f, 0x4f, 0x45, 0x8a, 0xe3, 0xc2, 0x4c, 0x04, 0x04, 0xc9, 0x99, 0x20, 0xb7,
+	0x53, 0xe6, 0x5a, 0x6b, 0x6c, 0x5d, 0x99, 0xee, 0x1d, 0xb0, 0x85, 0x0d, 0x0b, 0xee, 0x2a, 0xa0,
+	0x0d, 0x80, 0x8c, 0xf2, 0x7e, 0x4a, 0x49, 0xa4, 0xa4, 0x3e, 0xd0, 0x4a, 0xf7, 0x4e, 0x16, 0x0e,
+	0xe3, 0xc3, 0xb8, 0x83, 0xd7, 0x5d, 0xc9, 0x1e, 0x8c, 0x7d, 0x48, 0x62, 0x9c, 0x18, 0x4c, 0x28,
+	0xde, 0xc0, 0x2a, 0xcc, 0x3d, 0xa6, 0xa2, 0x79, 0xfc, 0x1c, 0xc6, 0x1e, 0xdf, 0xce, 0x8b, 0x62,
+	0xfd, 0xdd, 0x76, 0x5f, 0x42, 0xb2, 0x67, 0xb0, 0x42, 0x73, 0x93, 0xf1, 0x2f, 0x02, 0x98, 0xed,
+	0x71, 0x00, 0x2a, 0x99, 0x2a, 0x65, 0x93, 0x54, 0x82, 0xf5, 0x0c, 0x8f, 0x3f, 0x50, 0x51, 0x06,
+	0xf6, 0x35, 0xc4, 0x63, 0x06, 0x96, 0x14, 0xcf, 0x68, 0xcc, 0x5d, 0x34, 0x61, 0x3a, 0x5d, 0xde,
+	0x74, 0xe4, 0x68, 0x7f, 0xbb, 0x63, 0x86, 0x51, 0x7e, 0xe4, 0xf7, 0x5e, 0x30, 0xb9, 0xf5, 0x59,
+	0x8a, 0xa5, 0xc6, 0xe4, 0x01, 0x20, 0x04, 0x3e, 0xb4, 0xaf, 0xe5, 0x13, 0x86, 0x6a, 0xff, 0xeb,
+	0xab, 0x02, 0x03, 0x01, 0x00, 0x01,
+};
+
+const uint8_t test_rsa_pubkey_der_sig[256] = {
```

Comment:
Perhaps put this as a "static const uint8_t []" into some include boards/ssrc/saluki-xx, and use some define (BOARD_TEST_RSA_PUBKEY or such) as a pointer to it, then you can check if the macro is defined int the test_crypto.cpp and use either NULL or the pointer to the signature?

Reply:
Done. board specific change for NXP in https://github.com/tiiuae/saluki-nxp93/pull/34